### PR TITLE
support chromosome 23 and 24 in mutation mapper

### PR DESCRIPTION
**Pull request by leexgh**
Fix: https://github.com/cBioPortal/cbioportal/issues/7504

### CODE REVIEW

File: packages/cbioportal-utils/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "cbioportal-utils",
   "description": "cBioPortal Utilities",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (leexgh) [inline packages/cbioportal-utils/package.json:4]: @onursumer should I update version? I need this feature in transcript dropdown pr

Comment (onursumer) [reply packages/cbioportal-utils/package.json:4]: @leexgh Please revert this change. Package versioning is handled automatically. Once this PR is merged, `publish bot` will bump this version to `0.1.1`, take care of the dependencies, and publish new version(s) to npm.

File: packages/cbioportal-utils/src/mutation/MutationUtils.ts [hunk 2]
```diff
@@ -89,3 +93,13 @@ export function uniqueGenomicLocations(
 
     return _.values(genomicLocationMap);
 }
+
+export function normalizeChromosome(chromosome: string) {
```
Comment (onursumer) [inline packages/cbioportal-utils/src/mutation/MutationUtils.ts:97]: Apparently this is duplicate of this one: https://github.com/cBioPortal/cbioportal-frontend/blob/9bf8a68e8ad0b36d75da3a3b47bd4eeeff3dd50e/src/shared/lib/IGVUtils.ts#L93

Maybe move the already existing function here (I mean just copy the old implementation from `IGVUtils`, and delete the one there), and import the new function inside `IGVUtils.ts`.

File: packages/react-mutation-mapper/package.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
   "dependencies": {
     "autobind-decorator": "^2.1.0",
     "cbioportal-frontend-commons": "^0.2.10",
-    "cbioportal-utils": "^0.1.0",
+    "cbioportal-utils": "^0.1.1",
```
Comment (onursumer) [inline packages/react-mutation-mapper/package.json:42]: also revert this one. `publish bot` will take care of this, too.

File: packages/cbioportal-utils/src/mutation/MutationUtils.spec.ts [hunk 4]
```diff
@@ -164,6 +165,20 @@ describe('MutationUtils', () => {
                 'genomic location string should be:  1,100,100,C,G'
             );
         });
+        it('normalize chromosome and extract genomic location', () => {
```
Comment (onursumer) [inline packages/cbioportal-utils/src/mutation/MutationUtils.spec.ts:168]: Thanks for adding a test! Apparently there is no test for the existing function `IGVUtils.normalizeChromosome`. So, at least the test is not duplicate :)

By the way, I think this should read:
```suggestion
        it('normalizes chromosome and extracts genomic location', () => {
```

File: package.json [hunk 5]
```diff
@@ -133,7 +133,7 @@
     "bundle-loader": "^0.5.4",
     "cbioportal-frontend-commons": "^0.2.10",
     "cbioportal-ts-api-client": "^0.9.0",
-    "cbioportal-utils": "^0.1.0",
+    "cbioportal-utils": "^0.1.1",
```
Comment (onursumer) [inline package.json:136]: Can you revert this one, `publish bot` will take care of this.

File: packages/cbioportal-utils/package.json (hunk 1 above)
Comment (onursumer) [reply packages/cbioportal-utils/package.json:4]: Well... publish bot is broken due to the latest yarn version update. We need to merge cBioPortal/cbioportal-publisher/pull/2 before merging this PR

Comment (leexgh) [reply packages/cbioportal-utils/package.json]: @onursumer thanks for the explanation

**Comment by onursumer** [comment]
Unrelated e2e test is failing, merging

**Pull request merged**